docs/src/contributing/development-environment.md

```diff
@@ -151,42 +126,7 @@ The `deis` client is also written in Go. Your Deis client should match your serv
 
 ## Start Up a Development Cluster
 
-Our host system is now configured for controlling a Deis cluster. The next thing to do is begin standing up a development cluster.
-
-When developing locally, we want deisctl to check our local unit files so that any changes are reflected in our Deis cluster. The easiest way to do this is to set an environment variable telling deisctl where to look. Assuming the variable `$DEIS` points to the location if the deis source code, we want something like this:
-
-    $ export DEISCTL_UNITS=$DEIS/deisctl/units
-
-To start up and configure a local vagrant cluster for development, you can use the `dev-cluster` target.
-
-    $ make dev-cluster
-
-This may take a while to run the first time. At the end of the process, you will be prompted to run `deisctl start platform`. Hold off on that task for now. We will come back to it later.
-
-To verify that the cluster is running, you should be able to connect to the nodes on your Deis cluster:
-
-    $ vagrant status
-    Current machine states:
-
-    deis-01               running (virtualbox)
-    deis-02               running (virtualbox)
-    deis-03               running (virtualbox)
-
-    $ vagrant ssh deis-01
-    Last login: Tue Jun  2 18:26:30 2015 from 10.0.2.2
-     * *    *   *****    ddddd   eeeeeee iiiiiii   ssss
-    *   *  * *  *   *     d   d   e    e    i     s    s
-     * *  ***** *****     d    d  e         i    s
-    *****  * *    *       d     d e         i     s
-    *   * *   *  * *      d     d eee       i      sss
-    *****  * *  *****     d     d e         i         s
-      *   *****  * *      d    d  e         i          s
-     * *  *   * *   *     d   d   e    e    i    s    s
-    ***** *****  * *     ddddd   eeeeeee iiiiiii  ssss
-
-    Welcome to Deis     Powered by CoreOS
-
-With a dev cluster now running, we are ready to set up a local Docker registry.
+TODO: Document some approved "zero to kubernetes" options.
```

Comment:
I think there are a few different angles we need to attack in the docs.

For folks who are new to k8s, I think we need to recommend the best (easy and quick) way to get a kubernetes to try Deis. I get a little nervous, sending folks into the gaping maw of upstream `kube-up.sh` without at least a recommendation that they use a given tag. On any given day master can be sketchy. To me, this feels like hey, check out micro-kube, or kube solo, whatever the stable, user-friendly option is. Hoping folks make it through `insecure-registry` configuration is a really nasty cliff.

For folks who are ready to deploy, I think we can also have an opinion about the attributes or properties of a production kubernetes cluster. Here, we can lean our experience to say GKE or kube-aws on AWS is a good way to go, with these extensions enabled, etc.

So, with that context, we don't need to pull in step by step documentation for the zero to k8s matrix, but I think we need to offer a high-level guide so folks make it out of the forest.